src/backend/distributed/shared_library_init.c

```diff
@@ -1830,6 +1830,18 @@ RegisterCitusConfigVariables(void)
 		GUC_UNIT_MB | GUC_STANDARD,
 		NULL, NULL, NULL);
 
+	DefineCustomIntVariable(
+		"citus.max_parallel_tasks_per_node",
```

Comment:
bikeshedding: What should we call this setting? I think `citus.max_parallel_background_tasks_per_node` is more correct. It is quite long but maybe that's fine. In the design doc I called this `citus.max_parallel_moves_per_node`, but since it's now more generic I like your proposed name better.

@mulander what do you think?